tools/autograd/gen_autograd_functions.py

```diff
@@ -150,13 +234,18 @@ def process_function(info: DifferentiabilityInfo, template: CodeTemplate) -> str
 
     def save_var(var: SavedAttribute, is_output: bool) -> None:
         name = var.name
+        should_append_getsetdef = True
+
+        get_prop = GET_PROP.substitute(op=info.op, name=name)
+        get_prop_optionalarray = GET_PROP_OPTIONAL_ARRAY.substitute(op=info.op, name=name)
```

Comment:
while this is used twice, I think it is fine to move it down to simplify readability.